ExampleMVVM/Presentation/MoviesScene/MoviesList/ViewModel/MoviesListViewModel.swift

```diff
@@ -156,11 +155,25 @@ extension DefaultMoviesListViewModel {
         closures?.closeMovieQueriesSuggestions()
     }
 
-    func didSelectItem(at index: Int) {
-        closures?.showMovieDetails(pages.movies[index])
+    func didSelectItem(at indexPath: IndexPath) {
+        closures?.showMovieDetails(pages.movies[indexPath.row])
+    }
+}
+
+// MARK: - OUTPUT. View event methods
+
+extension DefaultMoviesListViewModel {
+
+    func item(for indexPath: IndexPath) -> MoviesListItemViewModel {
```

Comment:
```suggestion
    func item(for indexPath: Int) -> MoviesListItemViewModel {
```

#### Motivation
Using the `indexPath` the ViewModel is aware that the view is a list. What if the view was a grid or something else? IMO, Using an Int here, the ViewModel becomes more agnostic and with better relationships.
What do you think?

Reply:
Yes, can be Int. Especially because we do not use here sections. This is just for example App to show that we can have sections here too.